### PR TITLE
added module loading messages

### PR DESCRIPTION
**Pull request by usernameak**

### CODE REVIEW

**Comment by sojalei** [comment]
maybe need use logger

**Comment by evgfilim1** [comment]
Couldn't you use `.format()` instead of `'%s'`

**Comment by OctoNezd** [comment]
@evgfilim1 %s > format

**Comment by usernameak** [comment]
@OctoNezd +

**Comment by evgfilim1** [comment]
Sorry, but you have to use it to follow code standard (I don't like when styles are mixed)

**Comment by OctoNezd** [comment]
@evgfilim1 then switch to %!

**Comment by usernameak** [comment]
@evgfilim1 but... the logger supports additional args for %s

**Comment by evgfilim1** [comment]
@OctoNezd @usernameak Okay, I'll fix it myself, if you don't want to.

File: modloader.py [hunk 1]
```diff
@@ -1,32 +1,42 @@
 from os import listdir
 import api
+import logging
 
 ENABLED = {}
 DISABLED = []
 FAILURE = {}
 
 
+
```
Comment (evgfilim1) [inline modloader.py]: This newline is extra

File: modloader.py [hunk 2]
```diff
@@ -1,32 +1,42 @@
...
 def load_modules(updater):
     telegram_api = api.TelegramAPI(updater)
 
     for module_name in listdir('./modules'):
+
         if module_name.endswith('.py'):
             module_name = module_name[:-3]
+        else: continue
```
Comment (evgfilim1) [inline modloader.py]: `continue` must be on a new line

File: modloader.py [hunk 3]
```diff
@@ -1,32 +1,42 @@
...
+            logging.getLogger("modloader").info("Loading module '%s'... ", module_name)
             current_module = getattr(__import__('modules.{0}'.format(module_name)), module_name)
             loaded_module = current_module.TelegramModule(telegram_api)
         except Exception as e:
-            FAILURE.update({module_name: e})
+            logging.getLogger("modloader").warn("Cannot load module '%s': %s", module_name, str(e))
+            FAILURE.update({module_name: str(e)})
             continue
 
+
```
Comment (evgfilim1) [inline modloader.py]: Another extra newline

**Review by evgfilim1 — CHANGES_REQUESTED** [review]
I can do this myself, but you should know about these changes

File: modloader.py (hunk 2 above)
Comment (evgfilim1) [reply modloader.py]: And module can be a package (directory, which contains `__init__.py` file)

**Review by evgfilim1 — APPROVED** [review]

**Comment by OctoNezd** [comment]
WHAT HAVE YOU DONE YOU IMBECIL

**Comment by usernameak** [comment]
@OctoNezd сам имбицил

**Comment by OctoNezd** [comment]
@usernameak я про ебгена

**Comment by usernameak** [comment]
@usernameak кааааааааааааааааааааак?

**Comment by senegalha** [comment]
@usernameak Tem o código dele em algum Git?

**Comment by evgfilim1** [comment]
@dansenpir English, please

**Pull request merged**